index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @format
+ */
+
+import {AppRegistry} from 'react-native';
```

Comment:
I saw `eslint` as a dependency so you might borrow the rules of the extension codebase.
...this give me to think that we also could create a repository for eslint configuration and share it across projects.
Yet another point to discuss when we'll speak about splitting the apps @achauve @JalilArfaoui

Reply:
I mentioned `eslint` but I should also mention, prettier, the idea is of course to let the machine do the formatting and to format this file with a consistent code style.

Reply:
Ok I will apply eslint extension for mobile. I'm not sure getting very well what you mean with prettier? Do you think we should remove it or .gitignore it?

Reply:
Also, using TypeScript instead of FlowType would keep us closer to `extension`